BasicClasses/BasicClasses.tex

```diff
@@ -513,6 +513,7 @@ \section{Numbers}
 \centerline {\includegraphics[width=8cm]{NumberHierarchy}}
 \caption{The Number Hierarchy.\label{fig:numbers}}
 \end{figure}
+% TODO: ct: insert ScaledDecimal under number (right to Integer)
```

Comment:
@codeZeilen Can you do that? Or instruct me what software to use for graffle files? :)

Reply:
We will don't have access to OmniGraffle ;) Any figurers we adapt, we have to redo (currently I use Affinity Designer, as it is available on all platforms and not as expensive as Omni Tools). Also, whenever I redo a figure, I also store it in SVG, so we have a backup in case Affinity Designer also becomes expensive.

I would skip that change for now, or do you think it is absolutely necessary to update the number hierarchy?

Reply:
It looks a bit inconsistent right now, but if you deem it too expensive, I guess we can also skip that for now ... Maybe create a todo comment and change the caption to "The number hierarchy (excerpt)"?

Reply:
> Maybe create a todo comment and change the caption to "The number hierarchy (excerpt)"?

Sounds good. Are you on it or shall I? :)

Reply:
On it

Reply:
Btw, the title/sentence case we use in figures seems currently inconsistent ...